hack/examples/bases/nutanix/cilium/crs/kustomization.yaml.tmpl

```diff
@@ -0,0 +1,28 @@
+# Copyright 2024 D2iQ, Inc. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
I think this would be a good separate PR, just the changes needed to create example files and create the ClusterClass in the bootstrap. Without any of the API changes or handlers.

Reply:
https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pull/473